griptape/tools/image_gen/tool.py

```diff
@@ -0,0 +1,46 @@
+import os
+import openai
+from griptape.artifacts import BlobArtifact, ListArtifact, TextArtifact
+from griptape.tools import BaseTool
+from griptape.utils.decorators import activity
+from schema import Schema, Optional
+import requests
+
+
+class ImageGenerator(BaseTool):
```

Comment:
We should match the pattern throughout other tools and define an `attrs` class using the @attrs.define decorator here.